.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: test-substrate-client
+
+on:
+  push:
+    branches: [ testing ]
```

Comment:
This should be on all branches because we always wanna run the tests when someone changes code on any branch or create a new PR.